regression/goto-unwind/chain.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+
+goto_cc=$1
+goto_unwind=$2
+cbmc=$3
+is_windows=$4
+
+name=${*:$#}
+
+if [[ x$name == x ]];  then
+  name=${name%.c}
+fi
+
+args=${*:5:$#-5}
+
+if [[ "${is_windows}" == "true" && x$name != x ]]; then
```

Comment:
:grey_question: What is `x$name`?

Reply:
This is a really hacky way to check if `$name` is empty (the problem being that this otherwise might not parse correctly). This is basically saying "is x with $name appended still x?".

I've removed these cases because they’re not really needed here (I've taken this script from `goto-instrument` and just changed some bits).

Reply:
Ah, that makes sense. Thanks for the simplification.